docs/modules/ROOT/pages/explanation/synthesize.adoc

```diff
@@ -0,0 +1,122 @@
+Display Name:: Choose a descriptive name.
+Check first on the cluster listing overview how other clusters of the tenant are named.
+Lieutenant Tenant:: Choose the tenant to which this cluster belongs
+Service Level (Fact: service_level):: Check with Service Manager which Service Level applies to this cluster
+Cloud:: Check folder `cloud` in the {commodore-defaults-repo} for already defined clouds.
++
+Use value `none` for on-premise setups.
++
+If you set this field to `<name>`, there must be a file `cloud/<name>.yml` in the {commodore-defaults-repo}.
+See the how-to on xref:how-tos/create-missing-cloud-dist.adoc#_create_a_missing_cloud[creating a missing cloud] for some notes on adding a new cloud.
+Region:: Assuming that the value of field "Cloud" is `<cloudname>`, check folder `cloud/<cloudname>` in the {commodore-defaults-repo} for already defined regions for the cloud.
++
+Field "Region" is not required to be set for "Cloud" `none`.
++
+If you set this field to `<region>`, there must be a file `cloud/<cloudname>/<region>.yml` in the {commodore-defaults-repo}.
+See the how-to on xref:how-tos/create-missing-cloud-dist.adoc#_create_a_missing_region[creating a missing cloud region] for some notes on adding a new region to a cloud.
+Distribution:: Check folder `distribution` in the {commodore-defaults-repo} for already defined K8s distributions.
++
+If you set this field to `<dist>`, there must be a file `distribution/<dist>.yml` in the {commodore-defaults-repo}.
+See the how-to on xref:how-tos/create-missing-cloud-dist.adoc#_create_a_missing_region[creating a missing distribution] for some notes on defining a new K8s distribution.
+
+Rancher Cluster ID (Fact: rancher_cluster_id):: If it's a Rancher cluster, add the cluster ID from the Rancher Management Server to this field
+Repository Type:: Leave at "auto" to have Lieutenant manage the cluster catalog repo on https://git.vshn.net
+GitRepo URL:: Leave empty if "Repository Type" is auto.
+Check with the {projectsyn-ig}, if you have special requirements for the cluster catalog repo.
+
+[NOTE]
+====
+Not all combinations of attributes work smoothly.
+If in doubt, seek help from the {projectsyn-ig}.
+====
+
+== Running Commodore and adding the initial cluster configuration
+
+After running Commodore you have a local environment where you can edit your cluster's configuration.
+You can add initial configuration in the `<cluster-id>.yml` file in the tenant's Project Syn configuration repository (often called the "Syn tenant repo").
+The Syn tenant repo is cloned to `inventory/classes/<tenant-id>` by Commodore.
+
+See {portal-lieutenant-tenants} for a list of tenants and their associated configuration repositories.
+
+== Adding required tokens to Vault
+
+NOTE: We plan to add a direct link to a cluster's secrets in Vault to the Clusters view in {portal-lieutenant-clusters}, see Jira ticket https://ticket.vshn.net/browse/PORTAL-10[PORTAL-10].
```

Comment:
It looks like this will be already done very soon... See comments in PORTAL-10

Reply:
Updated to only note that the link to the cluster's secrets exists on control.vshn.net